src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day--tue calendar--month-length--28">
```

Comment:
```suggestion
      <div class="calendar calendar--tue calendar--28">
```
start-day и month-length это просто в задании указано за что отвечают эти модификаторы, их не обязательно писать здесь